src/app/dropdown/dropdown.component.css

```diff
@@ -131,6 +131,12 @@ hr {
     padding-top: 10px;
 }
 
+@media screen and (max-width: 2560px) {
```

Comment:
donot put a max width on this. Change this to min width form. Let us extend it to even beyond 2560px.
At 2670px:
![image](https://user-images.githubusercontent.com/20185076/29239460-cf7ae668-7f6c-11e7-92f8-75131326b86f.png)